Project/ISVR/src/jeet201801232.py

```diff
@@ -0,0 +1,197 @@
+import psutil
+import os
+import webbrowser
+import random
+import string
+from sys import platform
+import psutil as p
+from va import speak,ask

```

Comment:
src.va as the os.cwd would be outside the  folder so you would not be able to reference this correctly, see nmn.py

Reply:
after this, we're good to merge.